.github/workflows/compare_script_to_course.yml

```diff
@@ -0,0 +1,59 @@
+name: Compare SCT Commands
+
+on:
+  workflow_dispatch:
+    inputs:
+      text_url:
+        description: "URL to text file (e.g. GitHub raw gist link)"
+        required: true
+        type: string
+
+jobs:
+  compare:
+    runs-on: macos-latest
```

Comment:
Is there a reason for using macos? There's nothing wrong with it, and it matches the computer that's likely going to run the course; I guess I just have linux as a mental default.